src/lib/Libifl/pbs_loadconf.c

```diff
@@ -1109,11 +1112,14 @@ __pbs_loadconf(int reload)
 
 	pbs_conf.loaded = 1;
 
-	if (parse_psi(psi_value) == -1) {
-		fprintf(stderr, "Couldn't find a valid server instance to connect to\n");
-		free(psi_value);
-		goto err;
-	}
+	if (!reload) {
+		if (parse_psi(psi_value) == -1) {
+			fprintf(stderr, "Couldn't find a valid server instance to connect to\n");
+			free(psi_value);
+			goto err;
+		}
+	} else
+		fprintf(stderr, "Warning: PBS_SERVER_INSTANCES will not be re-read from pbs.conf, restart to take effect\n");
```

Comment:
I feel this is something to be handled at the daemon level. Some may choose to deal with it differently. Leave this function common for all daemons and let the daemons handle it the way it wants.

Reply:
but it is common, no daemon wants this param to be reloaded. In the future we might enhance them to be able to handle real time changes to PBS_SERVER_INSTANCES, but in the future PBS_SERVER_INSTANCES might not even exist, so let's not plan for that now.

Reply:
But how do you decide that? pbs_loadconf is an IFL.

Also, Are we throwing this warning on every reload irrespective of whether conf has PSI set or not? And what if PSI hasn't changed?

Reply:
"Also, Are we throwing this warning on every reload irrespective of whether conf has PSI set or not? And what if PSI hasn't changed?"

I can add checks for that.